presto-hive/src/main/java/com/facebook/presto/hive/HiveSplitManager.java

```diff
@@ -379,27 +382,32 @@ public CounterStat getHighMemorySplitSource()
                     partitions);
 
             ImmutableList.Builder<HivePartitionMetadata> results = ImmutableList.builder();
+            Map<String, Set<String>> partitionsNotReadable = new HashMap<>();
+            int unreadablePartitionsSkipped = 0;
```

Comment:
You should update `unreadablePartitionsSkipped` in the for loop body

Reply:
Sorry, I pushed the wrong commit.